src/docs/development/platform-integration/c-interop.md

```diff
@@ -0,0 +1,373 @@
+---
```

Comment:
@Vadaski Would like to know if this is the new doc? If so, we should add this doc to the translation task pool. https://github.com/cfug/flutter.cn/wiki/Keep-updated-with-the-official-site#%E5%A4%84%E7%90%86%E6%96%B0%E5%A2%9E%E7%9A%84%E6%96%87%E6%A1%A3-git-new-file
@hayley1643 could you please help with this, too?

Reply:
Please hold until I translate the title for this doc.
@Vadaski @hayley1643

Reply:
I've translated the title, please going forward.

Reply:
Yes. It's a new doc. We should add this doc to the pool.